streaming/dataset.py

```diff
@@ -1082,7 +1089,7 @@ def evict_coldest_shard(self) -> None:
         with self._cache_filelock:
             self._evict_coldest_shard()
 
-    def prepare_shard(self, shard_id: int, blocking: bool = True) -> None:
+    def fetch_shard(self, shard_id: int, blocking: bool = True) -> None:
```

Comment:
doc string of arguments missing

Reply:
fetch_shard has its args in its docstring?